### DISCUSSION TITLE
Gogs 0.12.6 has been released!

### DISCUSSION THREAD

**Discussion by unknwon** [gogs/gogs#6859] [Announcements]
Hello everyone,

Hope you have pleasant experiences with Gogs so far, and we’re excited to announce that Gogs 0.12.6 is released!

You can find binary downloads on our [website](https://gogs.io/docs/installation/install_from_binary) and the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.6) page.

The CHANGELOG for this release is also available on the [GitHub release](https://github.com/gogs/gogs/releases/tag/v0.12.6) page.

Please enjoy and happy coding!

Please comment on this thread if you spot any packaging issues for the release. For general feature requests and bug reports, please use our [curated issue forms](https://github.com/gogs/gogs/issues/new/choose) :)

### Backwards compatibility

You may upgrade your installation directly to this version if you're running Gogs 0.8.5 or newer.

**Comment by KrisztianHalasz** [comment]
Somehow these 2 files seems to be different and downloading from https://gogs.io/docs/installation/install_from_binary (Windows) marked as virus but github version looks OK.

https://dl.gogs.io/0.12.6/gogs_0.12.6_windows_amd64.zip
https://github.com/gogs/gogs/releases/download/v0.12.6/gogs_0.12.6_windows_amd64.zip

**Reply by unknwon** [reply]
Hmm, interesting, I downloaded both of them and they have identical SHA256 checksum:

```
→ shasum -a 256 gogs_*
7cb3284e12972a36decaa387192736c36f3cdb9d66ac738c0eab42b8f4715ea6  gogs_0.12.6_windows_amd64 (1).zip
7cb3284e12972a36decaa387192736c36f3cdb9d66ac738c0eab42b8f4715ea6  gogs_0.12.6_windows_amd64.zip
```

**Reply by KrisztianHalasz** [reply]
Thank you for checking, you are right. 
So I don't know what is our company protection doing and why. (facepalm)

![image](https://user-images.githubusercontent.com/2221260/163129964-678f0c07-99b4-43aa-bb6c-575e603041f2.png)
